modules/ossm-understanding-versions.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// * service_mesh/v1x/upgrading-ossm.adoc  ???
+// * service_mesh/v2x/upgrading-ossm.adoc
+// * service_mesh/v2x/ossm-troubleshooting.adoc
+
+[id="ossm-versions_{context}"]
+= Understanding Service Mesh versioning
+
+The {ProductName} 2.0 release introduced some confusion around version numbers because the 2.0 Operator supports both v1 and v2 service meshes.
+
+//OLM does not manage the upgrade
+//as a result, version number for the operator and a separate version number for the SMCP
+
+* *Operator* version - the most current Operator version is {ProductVersion}.  This version number only indicates the version of the currently installed Operator.  This version number is controlled by the intersection of the *Update Channel* and *Approval Strategy* specified in your Operator Subscription. The version of the Operator does not determine which version of the `ServiceMeshControlPlane` resource is deployed.   Upgrading to the 2.0 Operator does *not* automatically upgrade your control plane to v2.
+
+* *ServiceMeshControlPlane* version - The same Operator supports both v1.1 and v2.x control planes.  The SMCP version controls the architecture and configuration settings that will be used to install and deploy {ProductName}. To set or change the SMCP version you must deploy a new control plane. When you create the SMCP you can select the SMCP version in one of two ways:
+
+** To configure in the Form View, select the version from the *Control Plane Version* menu. Supported versions are v1.1 and v2.0.
+
+** To configure in the YAML View, set the value for `spec.version` in the YAML file.  The default is v2.0, but v1.1 is supported.
+
+* *Control Plane* version - The version parameter specified within the SMCP resource file as `spec.version`.  Supported versions are v1.1 and v2.0.
```

Comment:
The Form View is a GUI version of the YAML control plane file, sort of a wizard where you can select options rather than configuring them in the YAML code view.